tests/bootstrap.php

```diff
@@ -57,3 +57,5 @@
 Plugin::load('Auth', [
     'path' => dirname(dirname(__FILE__)) . DS,
 ]);
+
+$_SERVER['PHP_SELF'] = '/';
```

Comment:
This is reasonable to me. In a web context `PHP_SELF` would not be a file path to vendor/

Reply:
My git-fu isn't strong enough to sort out how to resolve this conflict. :-(

Reply:
Should have some time to get back to this on the weekend, now that a big deadline on another project is past. Anyone have suggestions on what git commands I need to run to merge the other commits into my own fork, resolve the conflict, and push back into my PR?

Reply:
Run `git merge upstream/master` (assuming this repo is named `upstream`). It will say `bootstrap.php` has conflict.

Open `bootstrap.php`, near end it will have
```
<<<<<<< master
Plugin::load('Auth', [
    'path' => dirname(dirname(__FILE__)) . DS,
]);

$_SERVER['PHP_SELF'] = '/';
=======
Plugin::getCollection()->add(new \Authentication\Plugin());
>>>>>>> master
```

Resolve conflict as 
```
Plugin::getCollection()->add(new \Authentication\Plugin());

$_SERVER['PHP_SELF'] = '/';
```

Now add updated file, commit and push to your fork as you always do (ex. `git add bootstrap.php`, `git commit`, `git push`, or use GUI tools)

Reply:
Awesome, thanks! I'm sure I tried something close to that, but obviously didn't hit the right combo. I mostly work in a "team" of one, so my git usage tends to be pretty straight-line.

Reply:
You've resolved conflict incorrectly...

In #233, the `Plugin::load(...);` was replaced with `Plugin::getCollection()...`. However you left both of them.

Your PR adds `$_SERVER['PHP_SELF'] = '/';` after that line.

Remove `Plugin::load(...);`. After that, _Files changed_ tab above should only show the line you've added. 

Also there is no new line now after `$_SERVER['PHP_SELF'] = '/';`.

Reply:
Yep, that was a foolish oversight, skipped through too quickly specifically because it was such a small change. :-(